### PR TITLE
tests: add test for custom sg

### PR DESCRIPTION
**Pull request by jor2**
### Description

Create test for custom_sg example

https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/issues/349

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

Create a test that implements the custom_sg example.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: tests/other_test.go [hunk 1]
```diff
@@ -89,6 +89,28 @@ func TestRunAddRulesToSGExample(t *testing.T) {
...
+		Prefix:        "cust-sg",
+		ResourceGroup: resourceGroup,
+		ImplicitDestroy: []string{
+			"module.ocp_base.null_resource.confirm_network_healthy",
+			"module.ocp_base.null_resource.reset_api_key",
+		},
+		// Do not hard fail the test if the implicit destroy steps fail to allow a full destroy of resource to occur
+		ImplicitRequired: false,
+		TerraformVars: map[string]interface{}{
+			"ocp_version": ocpVersion2,
```
Comment (ocofaigh) [inline tests/other_test.go:106]: this feature is only available in OCP 4.14, so you need to change this to `ocpVersion1`

**Review by ocofaigh — CHANGES_REQUESTED** [review]
see comment

**Comment by jor2** [comment]
> see comment

passed locally

Screenshot 2024-02-09 at 14 56 40

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**